freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -52,6 +53,13 @@
             nickname = "listFreeIpaClustersByAccountV1")
     List<ListFreeIpaResponse> list();
 
+    @GET
+    @Path("/healthDetails")
```

Comment:
would /health be sufficient? why /healthDetails?

Reply:
renamed